cmd/main.go

```diff
@@ -64,11 +64,17 @@ func main() {
 		klog.Fatalf("error parsing metrics bind address %s : %v", metricsBindAddress, err)
 	}
 
+	nodeName := os.Getenv("MY_NODE_NAME")
+	if nodeName == "" {
+		klog.Fatalf("node name not set, please set the environment variable using the Downward API")
```

Comment:
It would be better to make this a command-line option, and for it to be optional.

Reply:
but now the reconciliation logic depends on the nodename and the installation yaml defines the downward API variable ... is there any case you see people may want to run without this ?